content/en/th3/400_ressources/380_technical-spec.md

```diff
@@ -27,6 +27,9 @@ Version: 3.3
   * Both ARM 64 and AMD 64 architectures are supported via different installers.
   * No additional software is needed.
 
+**Major and minor versions:** Thorium releases minor versions each 3 to 4 months, those are numbered with a dot and numbers after the main version number (e.g. 3.1.1). Those versions usually include bug fix, minor improvements and enhanced features but no major break. This technical specification page address the main version and all it's minor versions. In case of significative difference with a minor version, this will be informed in the corresponding section.
```

Comment:
Thorium releases minor versions every 3 to 4 months; those are numbered with a dot and numbers after the main version number (e.g., 3.1.1). Those versions usually include bug fixes and enhanced features, but no breaking changes. This technical specification page addresses the main version and associated minor revisions. If a minor version introduces a notable difference, it will be clearly indicated in the relevant section.